### PR TITLE
update sdk and several modules

### PR DESCRIPTION
**Pull request by Lctrs**
wine to lutris-6.4
wine-nine to 0.7

recipes has been adapted from https://github.com/flathub/com.valvesoftware.Steam.CompatibilityTool.Proton

### CODE REVIEW

**Comment by Lctrs** [comment]
Just tested with Magic: The Gathering Arena. Works well.

**Comment by gasinvein** [comment]
I think there is no one left with push permission to merge it, besides flathub admins.

File: net.lutris.Lutris.Runner.Wine.yml [hunk 1]
```diff
@@ -48,15 +68,46 @@ cleanup:
   - /lib/wine/*.def
   - /lib32/wine/*.def
 modules:
+  - name: jxrlib
```
Comment (gasinvein) [inline net.lutris.Lutris.Runner.Wine.yml:71]: Does it even need jxrlib? I think it's something specific to Valve's Wine, no?

Comment (Lctrs) [reply net.lutris.Lutris.Runner.Wine.yml:71]: I had a warning locally telling me that wine couldn't find jxrlib while building it, so I guess it's used somewhere.

**Pull request closed without merging**